framework/test/unit/modules/dpos_v2/endpoint.spec.ts

```diff
@@ -163,9 +165,20 @@ describe('DposModuleEndpoint', () => {
 
 	describe('getAllDelegates', () => {
 		describe('when input address is valid', () => {
+			const address1Str = address1.toString('hex');
+			const address2Str = address2.toString('hex');
+
+			const addresses = [address1Str, address2Str];
+
+			const delegateData1 = Object.assign(delegateData, { address: address1Str });
+			const delegateData2 = Object.assign(delegateData, { address: address2Str });
+
+			// CAUTION!
+			// getAllDelegates() returns data in random order
```

Comment:
I think it's not random, it's based on key, lexicographically. If it was giving different order every time we call then it maybe called random.